src/app/shell/details/information-page/workshop-about/workshop-about.component.html

```diff
@@ -6,4 +6,27 @@ <h5> Вартість </h5>
   <p *ngIf="workshop.price"> {{ workshop.price }} грн/ {{ workshop.isPerMonth ? 'місяць' : 'заняття' }} </p>
   <h5> Наявність ресурсу для надання освіти для осіб з ООП та осіб з інвалідністю </h5>
   <p> {{ workshop.withDisabilityOptions ? workshop.disabilityOptionsDesc : 'Відсутні' }} </p>
+
+  <h5>Дні/Години</h5>
+
+  <div *ngFor="let timeRange of workshop.dateTimeRanges">
+    <div fxLayout='row' fxLayoutAlign='start center' fxLayoutGap='1rem' 
+    fxLayout.xs="column" fxLayoutAlign.xs="center start">    
+
+      <div fxLayout='row' fxLayoutAlign='space-between'>
+        <div *ngFor="let day of days" class="toggle" fxLayout='row' 
+          fxLayoutAlign='center center'
+          [ngClass]="timeRange.workdays.includes(workingDaysReverse[day.value] | titlecase)
+          ? 'toggle-selected' : ''"
```

Comment:
ngClass= {'toggle-selexted': true/false logic}

Reply:
Thnx, fixed it)